src/pages/WorkloadDetails/WorkloadInfo/WorkloadPodLogs.tsx

```diff
@@ -86,11 +87,15 @@ const tailToolbarMargin = style({
 export default class WorkloadPodLogs extends React.Component<WorkloadPodLogsProps, WorkloadPodLogsState> {
   private loadPodLogsPromise?: CancelablePromise<Response<PodLogs>[]>;
   private loadContainerLogsPromise?: CancelablePromise<Response<PodLogs>[]>;
+  private readonly podLogsRef: any;
+  private readonly proxyLogsRef: any;
```

Comment:
This is not a blocking comment but either now, or in the next follow-up work I'd like to see some name changes because I think it's confusing as is.  Both of these are part of the pod logs.  The pod has two containers, the istio proxy container and the app container.  So for example, I'd like to see renaming like:
```
loadPodLogsPromise -> loadAppLogs
loadContainterLogsPromise -> loadProxyLogs
podLogsRef -> appLogsRef
proxyLogsRef -> same
```
You get the idea.  This should be consistently applied.

Reply:
I meant to change that, but forgot.

Reply:
that was a legacy name.